ONSdigital/response-operations-ui
# Multimode create survey

# What and why?

For the rollout of multimode the create survey and edit survey pages need to updated to include the new multimode option for a survey. There is also a number of usability improvements included in both of those pages. 
# How to test?
Deploy the survey service changes to your environment and test the create and edit survey pages to ensure functionality and usability is as expected. 
# Trello
https://trello.com/c/2IZvLNGn/2450-multimode-mvp-creating-and-editing-existing-survey-to-add-multimode
